seed/utils/ubid.py

```diff
@@ -0,0 +1,50 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+import buildingid.v2
+import buildingid.v3
+
+from django.contrib.gis.geos import GEOSGeometry
+
+
+def centroid_wkt(state):
+    """
+    This translates GIS data saved as binary (WKB) into a text string (WKT).
+    """
+    if state.centroid:
+        return GEOSGeometry(state.centroid, srid=4326).wkt
+
+
+def decode_ubids(qs):
+    filtered_qs = qs.exclude(ubid__isnull=True)
+
+    for property in filtered_qs.iterator():
+        try:
```

Comment:
Just wanted to make sure to point this out - the app is capable of decoding v2 and v3 UBID. 

It was mentioned that buildingid.v2 should be used, but I couldn't find any data with UBIDs of that format. I could only get my hands on a dataset using the v3 format. Let me know if there would be any issues in allowing both formats to be decoded.

Reply:
i think we should favor v3 even though getting data may be harder. I'll do some UAT on this feature and let you know if there are any issues.